src/services/navigationBar.ts

```diff
@@ -307,7 +307,18 @@ namespace ts.NavigationBar {
                 addNodeWithRecursiveChild(node, getInteriorModule(<ModuleDeclaration>node).body);
                 break;
 
-            case SyntaxKind.ExportAssignment:
+            case SyntaxKind.ExportAssignment: {
+                const expression = (<ExportAssignment>node).expression;
+                if (isObjectLiteralExpression(expression)) {

```

Comment:
Array?

Reply:
@sheetalkamat I think it can be a new feature, however, in the current implementation Array elements are not available in NavigationBar for both cases not only for default exported arrays. 
<img width="706" alt="Screenshot 2020-05-04 at 08 49 07" src="https://user-images.githubusercontent.com/509265/80939211-83b5a180-8de4-11ea-920f-b1fdcb4fb0b6.png">